pkg/plugin/processor/standalone/processor.go

```diff
@@ -0,0 +1,360 @@
+// Copyright © 2023 Meroxa, Inc.
```

Comment:
should we start updating these to 2024 for new PRs? this might be committed in December, but just putting it out there that we should keep it in mind

Reply:
You're right there! I'm not sure if we can automate that (or of it's worth it), but you're definitely right in that we should keep it in mind.

Reply:
FWIW, I have set up the header in Goland using [copyright profiles](https://www.jetbrains.com/help/go/copyright-profiles.html), which always uses the current year when I create a new file, it doesn't require you to change anything every year. Lmk if you're interested in details, I can help you set it up.